types/js-cookie/index.d.ts

```diff
@@ -110,5 +110,5 @@ declare const Cookies: Cookies.CookiesStatic & {
     noConflict?(): Cookies.CookiesStatic;
 };
 
-export = Cookies;
+export default Cookies;
```

Comment:
This is incorrect, `js-cookie` doesn't use a default export, it uses a CommonJS export. Please also read this section of the readme: https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Did you want to export the native JS module version of this package? This is done differently, look at the types of the `uuid` package for an example of how this is done.